scripts/ci/publish.ts

```diff
@@ -1082,7 +1091,7 @@ async function acquireLock(): Promise<() => void> {
   const lock = promisify(require('redis-lock')(client))
 
   // get a lock of max 15 min
-  const cb = await lock('prisma2-build', 15 * 60 * 1000)
+  const cb = await lock(`prisma2-build-${branch}`, 15 * 60 * 1000)
```

Comment:
We don't need to stop `main` and `integration/` branches. We can lock them separately to only have 1 publish at a time on each branch.

So only 1 publish at a time can happen on `main`
Given the `integration/1` `integration/2` branches, each of these are also separate and can release concurrently